inst/qml/TTestPairedSamples.qml

```diff
@@ -66,6 +66,7 @@ Form
 				childrenOnSameRow: true
 				CIField { name: "effSizeConfidenceIntervalPercent" }
 			}
+			CheckBox {name: "effectSizeSE"; label: qsTr("Standard error")}
```

Comment:
For all tests: it would be good to make the behavior the same for the location parameter and the effect size.

- For location parameters, we have now one CI checkbox; if `TRUE`, both SE and the CIs for the location parameter are shown
- For effect size, we have a CI checkbox and an SE checkbox, controlling whether or not to display CI or SE, respectively.

-> please make this consistent.

I am actually thinking that there is probably not a reason why we should not show the SE by default, so one option could be just getting rid of the checkbox and show the SE next to the point estimate both for the location parameter and the effect size, whether or not the user also wants to see the CIs.

Reply:
Also note that it should be `effectSizeSe`, not `effectSizeSE`